File: 00_Learn_The_Basics/Things_To_Know_In_A_Language/For_Loops/sum.cpp
// Printing sum of numbers from 1 to N

#include<iostream>
using namespace std;

int main()
{
    
    int n,s=0;
    cout<<"Enter the value of n "<<endl;;
    cin>>n;
    
    for(int i=1;i<=n;i++)
    {
        s=s+i;
    }
    cout<<"The sum is : "<< s;
}

File: 00_Learn_The_Basics/Things_To_Know_In_A_Language/For_Loops/Sum.java
// Printing sum of numbers from 1 to N

import java.util.*;

public class Sum
{
    public static void main(String[] args)
     {
        
    int n,s=0;
    Scanner in=new Scanner(System.in);

    System.out.println("Enter the value of n ");
    n=in.nextInt();
    
    for(int i=1;i<=n;i++)
    {
        s=s+i;
    }
    System.out.println("The sum is : "+ s);
}
}
